### DISCUSSION TITLE
Having issues installing Foundry in my vs code

### DISCUSSION THREAD

**Discussion by abrahamdominic** [Cyfrin/foundry-full-course-cu#1723] [Q&A]
![Screenshot 2024-05-16 150305](https://github.com/Cyfrin/foundry-full-course-f23/assets/33906187/bfce46d0-23df-40a9-afc4-046caf9e2a80)

![Screenshot 2024-05-16 150415](https://github.com/Cyfrin/foundry-full-course-f23/assets/33906187/c9e5af44-75f3-49ca-bc12-1f395c4f8a09)

what should i do..?

**Comment by EngrPips** [comment]
Hello @abrahamdominic, You don't install Foundry with the `forge init` command. Please follow this link, `https://book.getfoundry.sh/getting-started/installation,` to properly install Foundry.
